packages/bpk-component-phone-input/stories.js

```diff
@@ -32,15 +32,7 @@ const DIALING_CODE_TO_ID_MAP = {
 const getFlag = dialingCode => {
   const countryCode = DIALING_CODE_TO_ID_MAP[dialingCode];
   const url = `https://images.skyscnr.com/images/country/flag/header/${countryCode}.png`;
-  return (
-    <BpkImage
-      altText="Flag"
-      height={38}
-      width={50}
-      style={{ width: '100%' }}
-      src={url}
-    />
-  );
+  return <BpkImage altText="Flag" height={38} width={50} src={url} />;
```

Comment:
we should probably do the same change in the readme too now that the element doesn't need the extra style.